tech/languages/dotnet/dotnet.md

```diff
@@ -0,0 +1,40 @@
+---
+title: .NET
+subsection: dotnet
+section: tech-languages
+order: 2
+version: 7.8.4
+description: ".NET Core and Mono open source frameworks, and available IDEs."
```

Comment:
This would be the description that will be displayed at https://developer.fedoraproject.org/tech.html where it's worth mentioning the content as whole, including IDEs.

Reply:
Not really I think, as only the page with order `1` get's to supply the description.

Reply:
(I hope I'm not mistaken.)